datadog/api/dashboard_lists.py

```diff
@@ -0,0 +1,48 @@
+from datadog.api.resources import (
+    ActionAPIResource,
+    CreateableAPIResource,
+    DeletableAPIResource,
+    GetableAPIResource,
+    ListableAPIResource,
+    UpdatableAPIResource
+)
+
+class DashboardList(ActionAPIResource, CreateableAPIResource, DeletableAPIResource, GetableAPIResource,
+    ListableAPIResource, UpdatableAPIResource):
+    """
+    A wrapper around Dashboard List HTTP API.
+    """
+    _class_url = '/dashboard/lists/manual'
+
+    @classmethod
+    def get_dashboards(cls, id, **params):
+        """
+        Get dashboards for a dashboard list.
+
+        :returns: Dictionary representing the API's JSON response
+        """
+        return super(DashboardList, cls)._trigger_class_action('GET', 'dashboards', id, params)
+
```

Comment:
I think we should add a `PUT`method (.. `PATCH` ideally but it's better to be consistent with other api).
This would allow people to modify dashboards in a list in an atomic way, without callling delete first. It comes in handy when people have lists stored somewhere in source control and want to make sure Datadog resources are in sync with what they have.